app/components/task/extension-form.hbs

```diff
@@ -0,0 +1,78 @@
+<div class='extension-form__container-parent'>
```

Comment:
@ivinayakg Here as you can see in the first video when we try to select ETA before than the current ETA it gives an error message and the modal expands and gets a scroll, and then if I select again the date greater than current ETA, the error message doesn't disappear.

What you can do is implement something like 2nd video demonstration, the dates are disabled even if the user can't select a month or year too.

[Screencast from 2023-03-03 22-54-23.webm](https://user-images.githubusercontent.com/78433013/222787324-e2c13e0a-6f7a-4cc3-8fc6-807b7d53779b.webm)

[Screencast from 2023-03-03 22-57-23.webm](https://user-images.githubusercontent.com/78433013/222787582-903f08d5-06de-4e6e-9605-3a56a06e2ca1.webm)